docs/edgeql/funcops/datetime.rst

```diff
@@ -416,7 +417,7 @@ Date and Time
         ...   <cal::local_datetime>'January 1, 2019 12:00AM', 'HKT');
         {<datetime>'2018-12-31T16:00:00+00:00'}
 
-    Yet another way to construct a the :eql:type:`datetime` value
+    Another way to construct a the :eql:type:`datetime` value
```

Comment:
I removed "Yet" here, because it looked a bit cumbersome when adding a new option below.

Reply:
+1